<?xml version="1.0"?>

<launch>


  <node name="map_loader" pkg="map_loader" type="map_loader" output="screen">      
      
      <remap from="/current_pose" to="/pose_estimate"/>  
      <!-- <remap from="/local_to_world/gps" to="/carla/ego_vehicle/gnss/gnss1/fix"/>
      <param name="global_sensor_frame" value="ins"/> --> 
       <!-- r_Town03_hj_traffic -->
      
      <param name="osm_file_name" value="$(find map_loader)/maps/kiapi_highspeed_local_path_v1_lanechange_fix.osm"/> 
      <!-- <param name="osm_file_name" value="$(find map_loader)/maps/unist_outv1_path_vel.osm"/>  -->
      <!-- <param name="osm_file_name" value="$(find map_loader)/maps/r_Town03_hj_traffic.osm"/>  -->
      <!-- <param name="osm_file_name" value="$(find map_loader)/maps/unist_0424_test_oneway.osm"/>  -->
      <!-- <param name="osm_file_name" value="$(find map_loader)/maps/unist_outv1_path_finall.osm"/>  -->
      
      <!-- <param name="map_origin_lat" value="0.002330359333808985"/> 
      <param name="map_origin_lon" value="4.5103238454623815"/>   -->
      
      <!-- UNIST MAP  -->
      <!-- <param name="map_origin_lat" value="35.57083743"/> 
      <param name="map_origin_lon" value="129.188112768"/>  
      <param name="map_origin_att" value="55.494"/>  -->
      
      <!-- KIAPI -->
      <param name="map_origin_lat" value="35.6470671983"/> 
      <param name="map_origin_lon" value="128.401913578"/>  
      <param name="map_origin_att" value="25.797"/>  

      <!-- <param name="map_origin_lat" value="0.0"/> 
      <param name="map_origin_lon" value="0.0"/>        
      <param name="map_origin_att" value="0.0"/>  -->

      <param name="test_direction" value="1.0"/> 
      
     
      <param name="visualize_path" value="True"/> 
      <param name="map_fix" value="False"/> 
      <param name="map_road_resolution" value="1.0"/> 
      <!-- <param name="local_path_length" value="30.0"/>  -->
      <param name="weight_decay_rate" value="0.9955"/> 
      <param name="lane_weight_decay_rate" value="0.96"/> 
      <param name="continuious_global_replan" value="True"/> 

      <param name="local_path_scale" value="1.0"/> 
      <param name="min_local_path_length" value="15.0"/> 
      <param name="max_local_path_length" value="35.0"/> 
      
      

      
      
  </node>


  
</launch>
